test/Serilog.Enrichers.Sensitive.Tests.Unit/WhenMaskingPaths.cs

```diff
@@ -0,0 +1,39 @@
+using FluentAssertions;
+using Xunit;
+
+namespace Serilog.Enrichers.Sensitive.Tests.Unit;
+
+public class WhenMaskingPaths
+{
+    private const string Mask = @"***\";
+
+    [Theory]
+    [InlineData(@"C:\Users\Admin\Secret\File.dll", @"***\", false)]
```

Comment:
I would expect this to be _only_ the mask in the expected output. Right now it looks like you're keeping the directory separator which seems odd.

Reply:
Don't really understand what do you mean